config.js.example

```diff
@@ -101,4 +101,4 @@ const config = {
   ]
 };
 
-module.exports = config;
+module.exports = config;
```

Comment:
Please, insert the leading new line again. ([eol-last](https://eslint.org/docs/rules/eol-last) rule in ESLint)